README.md

```diff
@@ -247,12 +255,16 @@ let mut stream = sqlx::query("SELECT * FROM users")
 ```
 
 ```rust
-#[sqlx::FromRow]
-struct User { name: String, id: i64 }
+#[derive(sqlx::FromRow)]
```

Comment:
I see why you added `email` to the type. Can you fix the formatting (no space before colon or comma) and keep `name` as `name` instead of `username` ?